mor_api_services/basis.py

```diff
@@ -161,7 +161,7 @@ def do_request(
 
         if cache_timeout and method == "get" and not force_cache:
```

Comment:
force_cache klinkt juist dat je afdwingt dat ALTIJD de cache wordt gebruikt, ik denk dat disable_cache of refresh beter zou zijn

Als force_cache true is verwijder je het item uit de cache pool, maar er is niet direct een nieuw item beschikbaar. Hiermee kunnen andere cache gebruikers in de tussentijd geen gebruik maken van de cache en kunnen er onbedoeld meerdere hits ontstaan.